ledger-service/http-json/src/main/scala/com/digitalasset/http/util/ExceptionOps.scala

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.http.util
+
+object ExceptionOps {
+
+  implicit class ExceptionOps(val underlying: Throwable) extends AnyVal {
+    def description: String = getDescription(underlying)
+  }
+
+  def getDescription(e: Throwable): String = {
+    val name: String = getClass.getName
```

Comment:
```suggestion
    val name: String = e.getClass.getName
```
Are you sure this is not what you're trying to do? Otherwise I believe name would always be `com.digitalasset.http.util.ExceptionOps$`.

Reply:
@stefanobaghino-da good catch! Thanks!